src/components/FilterPanel/FilterPanelSearch/__tests__/FilterSearch.spec.js

```diff
@@ -19,24 +19,24 @@ describe('FilterPanelSearch', () => {
   });
 
   it('renders results content when keyboard focus is added', () => {
-    const { getByTestId, getByLabelText } = render(
+    const { getByTestId, getAllByLabelText } = render(
```

Comment:
@diego-codes Just bringing the updates in this file to your attention -- anything amiss?

Reply:
I suspect that the labels changed so `getByLabelText` errors out because there is more than one element with this label?

It might be worth switching the queries to `getByPlaceholderText` so you're not dependent on the order of elements that are labeled `"search label"`

Reply:
Thanks @diego-codes! Unfortunately `getByPlaceholderText` won't work here. 🤔 (I suspect because we are dealing with `label` elements, not placeholder attributes)

Reply:
Sorry, I meant that you can provide a placeholder text to the search component and query it with that placeholder text.

Reply:
Just committed the update 👍